subsys/emds/Kconfig

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2022 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig EMDS
+	bool
+	prompt "Emergency Data Storage"
+	default n
+	help
+	  Enable Emergency Data Storage.
+
+if EMDS
+
+config EMDS_SECTOR_SIZE_MULT
+	int "Sector size of the emergency data storage area"
+	default 1
+	help
+	  The sector size to use for the emergency data storage area as a
+	  multiple of FLASH_ERASE_BLOCK_SIZE.
+
+config EMDS_SECTOR_COUNT
+	int "Sector count of the emergency data storage area"
+	default 1
+	help
+	  Number of sectors used for the emergency data storage area
```

Comment:
Is there a need for defining sector bigger than `FLASH_ERASE_BLOCK_SIZE`?

Reply:
I removed the EMDS_SECTOR_SIZE_MULT as I do not think it is necessary.